docs/models.rst

```diff
@@ -0,0 +1,101 @@
+
+======
+Models
+======
+
+- :ref:`name-model-ref`
+- :ref:`identifier-type-model-ref`
+
+.. _name-model-ref:
+
+Name
+----
+Name objects have a variety configurable options. 
+
+Fields
+''''''
+
+``name`` - The canonical form of the name.
+
+``name_type`` - One of `Personal`, `Organization`, `Software`, `Building`, or `Event`
+
+``biography`` - Markdown enabled biography of the personal the Name record represents.
+
+``begin`` - The starting date for the Name record. This will be different for each Name Type, for instance, this field would be the birth date for a `Personal` name, and the erected date for a `Building` name.
```

Comment:
I don't think there should be a comma in here: "for a `Personal` name, and the erected date"